<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_noFocus">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>340</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="placeholderText">
    <string>无法聚焦</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tabFocus_1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>340</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="placeholderText">
    <string>Tab聚焦</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tabFocus_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>340</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="placeholderText">
    <string>Tab聚焦</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_clickFocus">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>340</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="placeholderText">
    <string>点击聚焦</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_strongFocus">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>340</width>
     <height>40</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Tab | 点击聚焦</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
